packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagHasPrimaryAction.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Popover, PopoverTrigger, Button, PopoverSurface, makeStyles } from '@fluentui/react-components';
+import { InteractionTag } from '@fluentui/react-tags-preview';
+
+const useStyles = makeStyles({
+  popover: {
+    width: '360px',
+    maxWidth: '100%',
+    height: 'fit-content',
+  },
+  popoverHeader: {
+    display: 'flex',
+    columnGap: '10px',
+    alignItems: 'center',
+  },
+});
+
+export const HasPrimaryAction = () => {
```

Comment:
This might not be working as intended? 
![interaction tag actino=](https://github.com/microsoft/fluentui/assets/20744592/2057dba0-fadd-4d3a-a8dc-44a4bb6d4c54)

Reply:
This is me using Enter/Space on the dismiss button, it seems weird that a click will dismiss but those keys will trigger the primary action